docs/core/integration-quality-scale/rules/iqs042.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "IQS042 - The documentation provides step-by-step installation instructions for the integration, including, if needed, prerequisites"
+---
+
+IQS042 is as follows:
+
+The documentation provides step-by-step installation instructions for the integration, including, if needed, prerequisites
+
+## Reasoning
+
+We want users to have a smooth experience when setting up an integration.
+This means that the documentation should provide clear and concise instructions on how to install the integration.
+This includes any prerequisites that are needed to install the integration.
+
+## Example implementation
+
+```markdown
+# Installation
+
+1. Install the app off the app store
+2. Create an account in the app
+3. Add a device in the app
+4. Go to the settings page in the app
+5. Click the "Expose API" button
+6. Continue at Home Assistant and add the integration
```

Comment:
```suggestion
## Prerequisites

1. Open the app store and install the **MyProduct** app.
2. Create an account.
3. Add a device to the app.
4. Open the app and go to the **Settings** page.
5. Select **Expose API**.

{% include integrations/config_flow.md %}
```

Renamed the Installation section to Prerequisites, as the configuration section is usually pulled in from the config flow. Or was this another use case here?